samples/downloadPublicFile.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Storage Download Public File.
+//   description: Download Public File.
+//   usage: node downloadPublicFile.js <BUCKET_NAME> <SRC_FILE_NAME> <DEST_FILE_NAME>
+
+const path = require('path');
+const cwd = path.join(__dirname, '..');
+
+function main(
+  bucketName = 'my-bucket',
+  srcFileName = 'test.txt',
+  destFileName = path.join(cwd, 'downloaded.txt')
+) {
```

Comment:
This sample should use an unathenticated client instead of http package directly.

Reply:
https://github.com/googleapis/nodejs-storage/pull/82/files

Reply:
we don't have unauthenticated client like in [JAVA](https://github.com/googleapis/google-cloud-java/blob/5203d14fbc3af58a9cae2838b908fec42ddf5f8d/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/objects/DownloadPublicObject.java#L38) I just modified the sample that explain how to remove authentication.

Reply:
@stephenplusplus is this a feature gap in the library?